### PR TITLE
Create 2024-07-03-csail-network

### LINKED ISSUE #1341: Post Status Update on massopen.cloud site
---
title: CSAIL network redesign may affect ESI and MAAS users on July 15, 2024
state: active
severity: 1

# Prevent template from getting published. Remove this
# line when copying the template.
published: false
---
CSAIL will make changes to its network configuration on July 15, 2024. This change may affect users of ESI and MAAS and may affect their public IP addresses.

### PR DESCRIPTION
**Pull request by msdisme**
Notification of possible ESI and MAAS service interruption on July 15, 2024

### CODE REVIEW

File: _posts/2024-07-03-csail-network [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+title: Changes to CSAIL network on July 15, 2024 may affect ESI and MAAS users
+state: active
+# Severity: 0=All systems are operating normally.
+# 1 = There are one or more service interruptions.
+# 2 or more = There is a critical service outage.
+severity: 1
+
+# Prevent template from getting published. Remove this
+# line when copying the template.
+published: false
```
Comment (joachimweyl) [inline _posts/2024-07-03-csail-network:11]: I think if you want it published you need to remove this line or add a # before it

File: _posts/2024-07-03-csail-network [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+3. Optionally set a `severity`. By default, posts have a severity of 1
+   (mild). You can also select 0 (low) or 2 (critical).
+
+4. Save this as _posts/YYYY-MM-DD-short-title.md.
+
+Marking an existing status entry as resolved:
+
+1. Remove the `state` setting from the frontmatter.
+
+-->
```
Comment (hakasapl) [inline _posts/2024-07-03-csail-network:35]: We don't need the template lines

**Review by hakasapl — CHANGES_REQUESTED** [review]
File is not saved as .md file

File: _posts/2024-07-03-csail-network [hunk 3]
```diff
@@ -0,0 +1,35 @@
+---
+title: Changes to CSAIL network on July 15, 2024 may affect ESI and MAAS users
+state: active
+# Severity: 0=All systems are operating normally.
+# 1 = There are one or more service interruptions.
+# 2 or more = There is a critical service outage.
+severity: 1
```
Comment (joachimweyl) [inline _posts/2024-07-03-csail-network:7]: severity 0 until 15th

**Comment by joachimweyl** [comment]
no longer needed

**Pull request closed without merging**